app/Models/User.php

```diff
@@ -465,4 +470,11 @@ public static function fetchEmails(): Collection
     {
         return self::where('subscribe', 1)->whereNotNull('email')->get()->pluck('email');
     }
+
+    public static function generateRandomETHWalletForSwash(): string {
```

Comment:
I think creating a normal wallet is more random than limiting chars and shortening the wallet length

Reply:
Right. The leading zeros are removed.